src/Query/Filter/DateField.ts

```diff
@@ -38,6 +38,7 @@ export abstract class DateField extends Field {
     }
 
     public canCreateFilterForLine(line: string): boolean {
+        line = line?.toLowerCase();
```

Comment:
The question mark is used to do Optional Chaining.
In this case it's useless because the line is never undefined, if I remember correctly I put it here because at some point I had a mistake and forgot to remove it.
Thanks for catching it, I will remove it :smile: